### DISCUSSION TITLE
why only three options are shown? when i use different efi folder install macos shows but when i use ux481 efi only 3 options were appear

### DISCUSSION THREAD

**Discussion by illuminatijattz** [Qonfused/ASUS-ZenBook-Duo-14-UX481-Hackintosh#27] [Install Help]
![FF8D10C0-947F-4914-97EC-91FFF38D89F3](https://user-images.githubusercontent.com/122349532/211510505-a5e16633-856b-47ab-8cf9-9cf526a8eeee.jpeg)

**Comment by Maleficent-Magik** [comment]
And if you press Space ?

**Reply by illuminatijattz** [reply]
try but nothing happens

**Reply by Maleficent-Magik** [reply]
I'm trying to understand, don't mind if I ask stupid questions ... but can you tell me :

-> When you say "when i use different efi folder install macos shows": Do you make a virgin key again? or do you replace each time?

-> Can you take a picture of the "Drivers" folder? 

And we agree that the name of the folder where the installation file is located is called "com.apple.recovery.boot"?

**Reply by illuminatijattz** [reply]
i tried both methods old and new 
i mean .raw file and .dmg file method by making com.apple.recovery.boot but have same issue each time with this efi files

**Reply by Qonfused** [reply]
Double check that you've set SATA mode in BIOS to `AHCI`. There are some troubleshooting steps [here](https://dortania.github.io/OpenCore-Install-Guide/troubleshooting/extended/opencore-issues.html#can-t-see-macos-partitions) that are worth double checking, but the default behavior should show the macOS installer.

**Reply by illuminatijattz** [reply]
> I'm trying to understand, don't mind if I ask stupid questions ... but can you tell me :
> 
> -> When you say "when i use different efi folder install macos shows": Do you make a virgin key again? or do you replace each time?
> 
> -> Can you take a picture of the "Drivers" folder?
> 
> And we agree that the name of the folder where the installation file is located is called "com.apple.recovery.boot"?

i put chunklist and dmg inside com.apple.recovery.boot folder 
and here is my drivers files 

![AAEA46C6-09B2-4E83-BAC0-8E8E72629A45](https://user-images.githubusercontent.com/122349532/212056798-7b13bcaa-746d-4ee3-8b7f-b3c881d0608f.jpeg)

**Reply by illuminatijattz** [reply]
> Double check that you've set SATA mode in BIOS to `AHCI`. There are some troubleshooting steps [here](https://dortania.github.io/OpenCore-Install-Guide/troubleshooting/extended/opencore-issues.html#can-t-see-macos-partitions) that are worth double checking, but the default behavior should show the macOS installer.

i set scan policy at 0 and usb drive option shows but after i click on it i have this error

![22D107D0-9E2B-4CBF-A4EB-E8B7C8CF7AC7](https://user-images.githubusercontent.com/122349532/212073155-a628fa8e-4d03-43d9-b5c5-fbbfa7b97cb6.jpeg)

**Reply by Qonfused** [reply]
You can try creating a new fat32 partition on your SSD (under any name) with your EFI and `com.apple.recovery.boot` folder, which should work for the sake of testing. I'd recommend moving it to the SSD's EFI partition after installing macOS (some instructions on doing that [here](https://dortania.github.io/OpenCore-Post-Install/universal/oc2hdd.html)). Would you mind sharing the other EFI that did work off USB? I'm curious as to what is causing the issue on USB.

**Reply by illuminatijattz** [reply]
HERE IS THE EFI FOLDER YOU ASK FOR 

https://drive.google.com/drive/folders/16jgPaprzsCx_aFDVbfJmzZES9lBHdSjZ

**Reply by Qonfused** [reply]
The `LauncherOption` value under `Misc > Boot` in the config.plist may be overriding the opencore boot option. You'll probably want to clear your NVRAM with the `Reset NVRAM` option from OpenCore. I'd also strongly advise against using Olaria.

If you encounter issues with the BIOS not recognizing the USB, I'd recommend booting into BIOS and double checking you've disabled the `Fast Boot` option under `Boot` and that you've enabled `USB Mass Storage Driver Support` under `Advanced > USB Configuration`. UEFI booting from USB should be working out of the box with these options enabled. You can also hold the Esc key while powering on to quickly change the boot device.

**Reply by illuminatijattz** [reply]
most of the part gone well but after first installation in recovery mode i select macos installer and this error showed.what could be the reason ?
![747A664F-AEA6-4823-9078-D69146E2AEFA](https://user-images.githubusercontent.com/122349532/212310739-8f180e6f-79a2-4ae9-9bc7-7df7071cc312.jpeg)

**Reply by Qonfused** [reply]
This kernel panic is caused by the SSD somehow (`com.apple.iokit.IONVMeFamily`). Check in the BIOS if you have `SATA Mode Selection` set as `AHCI` (under `Advanced > SATA Configuration`). If you have some sort of Intel Optane option in BIOS, you'd want to disable that option as well. If you are using a different SSD than the Intel 660p, what SSD are you using?

**Reply by illuminatijattz** [reply]
i have samsung nvme m.2 pcie ssd is there any way to install macos in this ssd

**Reply by Qonfused** [reply]
Do you know what model SSD? E.g. Samsung 970 Evo.

**Reply by illuminatijattz** [reply]
SAMSUNG MZVLB1TOHALR-00000 (EXA73010)  i think its samsung pm981

**Reply by Qonfused** [reply]
Unfortunately the pm981 drive won't work with macOS at all; it's actually explicitly mentioned [here](https://dortania.github.io/Anti-Hackintosh-Buyers-Guide/Storage.html) to cause these kernel panics. The Intel 660p drive that came with the zenbook duo would work, as would most other SSDs.

**Reply by illuminatijattz** [reply]
ok

**Reply by illuminatijattz** [reply]
intel 660p ssd read write speed is low 
can you suggest some supported high speed ssd

**Reply by Qonfused** [reply]
The best all-rounder is the Western Digital SN750; in general, their blue/black SSDs are the fastest w/ macOS.
